plugins/mcreator-localization/lang/texts.properties

```diff
@@ -1313,6 +1313,7 @@ blockly.block.world_send_chat=Send to chat to all players: %1 %2
 blockly.block.world_set_downfall=Enable rain if %1 otherwise, disable it
 blockly.block.world_spawn_falling_block=Spawn %1 as a falling block at x: %2 y: %3 z: %4 %5
 blockly.block.world_switch_world=Do %1 in server-side world %2
+blockly.block.world_trigger_gameevent=Trigger %1 gama event at x: %2 y: %3 z: %4 by %5 %6
```

Comment:
typo in "gama event"

Reply:
It would also read better to swap it a bit, e.g.

"Trigger game event %1 at ... "